### PR TITLE
Button

### PR DESCRIPTION
**Pull request by iamsadafmhmdi**

### CODE REVIEW

File: src/components/Button/index.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+button{
```
Comment (Arsalan74) [inline src/components/Button/index.css:1]: put a space after button 
like 
button {
do the same for all the other cases

Comment (iamsadafmhmdi) [reply src/components/Button/index.css:1]: just for css files?

File: src/components/Button/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './index.css'
+
+
+function Button ({disabled=false, value, handleClick}) {
```
Comment (Arsalan74) [inline src/components/Button/index.js:5]: You can give the text of the buttons by two means
you can give the text to it by giving the value prop of the button
or you can give it as children 
like this 
<button> {children} <button />
or
<button value={some text} />

File: src/components/Button/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    return(
+        <button className={disabled ? 'disabled' : undefined} onClick={handleClick}>{value}</button>
```
Comment (Arsalan74) [inline src/components/Button/index.js:7]: handleClick is used when we have useCallbacks (it should be the name of the useCallback)
but the so you should change the handleClick to onClick

File: src/App/index.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import Button from '../components/Button/index';
 
 
 function App() {
     return(
         <div>
-            App
+            <Button children={'click'} value={'click'}/>
```
Comment (Arsalan74) [inline src/App/index.js:8]: you should do it like this 
<button value={'click}> 
  'click'  //this is the children when you render button somewhere with children this "click appears"
<button />

File: src/components/Button/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+import './index.css'
+
+
+function Button ({disabled=false, children, onClick, value}) {
+    return(
+        <button className={disabled ? 'disabled' : undefined} onClick={onClick} value={value}>{children}</button>
+    );
+};
+
+export default Button;
```
Comment (Arsalan74) [inline src/components/Button/index.js:11]: add an empty line at the end of your line

**Pull request merged**